components/runtime/chezscheme/Makefile

```diff
@@ -0,0 +1,87 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Richard Lowe
+#
+PREFERRED_BITS=64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		chezscheme
+COMPONENT_VERSION=	9.5.2
+COMPONENT_SUMMARY=	Chez Scheme
+COMPONENT_PROJECT_URL=	https://github.com/cisco/ChezScheme
+COMPONENT_FMRI=		runtime/chezscheme
+COMPONENT_CLASSIFICATION=Development/Other Languages
+COMPONENT_SRC=		ChezScheme-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+  sha256:3a370fdf2ffd67d6a0ccbb993dfab1cbaf4a0a97983c869cfaab40528c33c48b
+COMPONENT_ARCHIVE_URL= \
+  https://github.com/cisco/ChezScheme/archive/v$(COMPONENT_VERSION).tar.gz
+
+# Nanopass
+COMPONENT_NAME_1= nanopass-framework-scheme
+COMPONENT_VERSION_1= 1.9
+COMPONENT_SRC_1= $(COMPONENT_NAME_1)-$(COMPONENT_VERSION_1)
+COMPONENT_ARCHIVE_1= $(COMPONENT_SRC_1).tar.gz
+COMPONENT_ARCHIVE_URL_1= https://github.com/nanopass/$(COMPONENT_NAME_1)/archive/v$(COMPONENT_VERSION_1).tar.gz
+COMPONENT_ARCHIVE_HASH_1=	\
+  sha256:625b239f9030d0b1e86b1fffd8b69f7249a63e8b8ca85195a00cf22889f7fc86
+
+# zlib (configure would get this from github.com/madler/zlib)
+# There seems to be no provision to use a system zlib
```

Comment:
Someone [here](https://build.opensuse.org/package/view_file/home:lorenz:idris/ChezScheme/use_native_zlib.patch?expand=1) may got it to work (or to build at least).